src/main/java/com/thedariusz/warnme/twitter/AuthorDto.java

```diff
@@ -1,7 +1,10 @@
 package com.thedariusz.warnme.twitter;
 
+import com.fasterxml.jackson.annotation.JsonProperty;
+
 public class AuthorDto {
 
+    @JsonProperty("author_id")
```

Comment:
Try to avoid using @JsonProperty if possible as this is like code pollution. 
If i am not wrong your current jackson configuration in spring boot already uses snake_case so you coud name this filed `authorId` and it will know how to map it